### ISSUE TITLE
Counter for blocked domains (like in Blokada 4)

### ISSUE THREAD

**Issue by qwerty287** [blokadaorg/blokada#850]
Hello,
in Blokada 4, there was the feature to see how many domains are blocked. Please add this to Blokada 5!

**Comment by peterroth** [comment]
Hi,
There is, on the main screen.
![Screenshot_20210311-211331__01.jpg](https://user-images.githubusercontent.com/35808401/110848841-d090c100-82ae-11eb-98e4-62847b707c0e.jpg)

**Comment by qwerty287** [comment]
No, I mean the counter for the domains that are in all blocklists together. In Blokada 4, this was in the settings of the ad blocker. So, a counter that includes all domains that would be blocked, not only the domains that have already been blocked.

**Comment by peterroth** [comment]
Oh, I see. Thank you for the clarification.
